### PR TITLE
269495 create jb

### PR DESCRIPTION
**Pull request by jbarnsley10**
Owner details page
ToDo - date validation only fires once other required fields are completed

### CODE REVIEW

**Review by sfitz42 — CHANGES_REQUESTED** [review]
Looks good - Just a couple of redundant constants that need to be removed.

File: app/session/register/owner.js
[the remark this answers is not in the document]
Comment (jbarnsley10) [reply app/session/register/owner.js:1]: Removed redundant ones - name + dateOfBirth

File: app/models/register/owner/owner-details.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+const { routes } = require('../../../constants/owner')
+
+function ViewModel (ownerDetails, errors) {
+  this.model = {
+    formAction: routes.ownerDetails.get,
+    backLink: routes.address.get,
```
[the remark this answers is not in the document]
Comment (jbarnsley10) [reply app/models/register/owner/owner-details.js]: Probably, although we don;t know the outside navigation yet. Will change it

File: app/constants/owner.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-      get: '/register/owner/name',
-      post: '/register/owner/name'
+    home: {
+      get: '/'
     },
-    postcode: {
-      get: '/register/owner/postcode',
-      post: '/register/owner/postcode'
+    ownerDetails: {
+      get: '/register/owner/owner-details',
```
Comment (sfitz42) [inline app/constants/owner.js:7]: Sorry just realised these should match the routes on the Mural board.

This one should be `cdo/create`.

Comment (jbarnsley10) [reply app/constants/owner.js]: Done

**Review by sfitz42 — APPROVED** [review]

**Pull request merged**